2-Python Introduction/solution.md

```diff
@@ -0,0 +1,154 @@
+ 
+   TYPE ()
+
+3-What is the difference between / and //?
+
+   use a / is for division to return decimal numbers
+
+   using two // in a division serves so that the answer of the division is in    whole numbers
+
+4- What does the function pow(a, b) do?
+
+    It is for calculating the power rise of a number. 
+    (a) is the base of the number that will rise to power
+    (b) is the exponent
+
+5- What does the function round(a, b) do?
+
+rounds to the given number of digits and returns the floating point number, if no number of digits to round is given, rounds the number to the nearest integer.
+
+6- What does the function bin(a, b) do?
+
+   Return the binary representation of an integer
+
+7- What does the function hex(a, b) do?
+
+   Returns the hexadecimal representation of an integer
+
+8- What does it mean that a variable is immutable?
+
+   means that if the value can change, the object is called mutable, while if the value cannot change, the object is called immutable
+
+9- If a=[0,1,2,3,4] and a[0]=0, what's the value of a[-1]?
+
+    the value is (4)
+
+10- What does the operator in do?
+
+    in returns True if the specified value is found in the sequence. Otherwise it     returns False.
+
+
+11- What is operator precedence?
+
+    Python operators have a set precedence order, which determines which operators       are evaluated first in a potentially ambiguous expression.
+
+
+12- How is different an expression from a statement? 1.What is an augmented operator? write a code block with an example.
+
+    declarations are anything that can make up a line (or lines) of Python code.     Note that expressions are also statements.
+
+    expressions contain only identifiers, literals, and operators, where operators     include arithmetic and boolean operators, the call operator function of () the     subscription operator [], and the like, and can be reduced to some kind of     "value", which can be any object
+
+
+13- What is string concatenation? write a code block with an example.
+
+    allow you to assign a value to a variable.  example:  r=5; r = r+10
+                                                          print(r)                                                                       
+                                                          answer: 15
+
+14- What is escape sequence and why is it used?
+
+      Concatenation can be defined as the integration of two strings into one       object.     example:  x = "El monto de mi cuenta de banco es de "
+                                                                                                                 y = 100 000
+                                                                                                                 print(x + str (y))     
+                             
+                                                                                                                answer: "El monto de mi cuenta de banco es de 100 000"     
+
```

Comment:
this is the answer of question above :v check question-answers match. Also use markdown for code. For example you can write 

example code:
```
x = "El monto de mi cuenta de banco es de "
y = 100 000
print(x + str (y))     
````

console output:
```
El monto de mi cuenta de banco es de 100 000
```